skycultures/indian/constellation_names.eng.fab

```diff
@@ -19,49 +20,85 @@
 # If your skyculture is not a variant of the generally known Western skyculture, please include an
 # English translation to the name given in the native language. Else translators will not be able to
 # translate the name
+#################
+
+# For the column 1: An abbreviation for the constellation. (The "abbreviated" mode should allow to use short unique markers, without filling the screen with text. In scientific context, the 88 IAU constellations are abbreviated with 3 characters, which is just enough to remember the constellation.)
+# For the column 2: We use IAST transliteration for unambious transliteration of devanAgarI using latin alphabets, while using phonetically more intuitive equivalents in suggested translations.
+# For column 3: We've referred to highly regarded sanskrit to sanskrit dictionaries kalpadruma and vAchaspatyam for etymologies below.
 
-# We use IAST transliteration for unambious transliteration of devanAgarI using latin alphabets, while using phonetically more intuitive equivalents in suggested translations.
+#################
+# Nakshatra-s.
+#################
+# We show Nakṣatra-s associated with a single star as a star, rather than as a constellation.
+# From @gzotti: "if some Nakshatras may receive sky art, all Nakshatras should be coded as constellations. When we do the change mentioned above, those will then be separated (or just marked inside the constellation file). ... We may highlight the 1-star constellation in later editions e.g. by a circle." 
 
-## We've referred to highly regarded sanskrit to sanskrit dictionaries kalpadruma and vAchaspatyam for etymologies below.
+Aśv	"Aśvinī"	_("Divine physician twins")
+Bhar	"Bharanī"	_("Bearer")
+Kṛt	"Kṛttikā"	_("Cutter")
```

Comment:
I see severely different translations for some names given (Ardra: moist or wet? Krttika??? phalguni=red (just the color), or fruitful, as in "ripe (-->red?) fruit"; chitra=just bright or now even spectacular?; svati=honest, or untranslatable proper name? Dhanista=wealthiest, or most famous???), and even a swap in sequence,  as e.g. former N05 Mrgasira has moved down. Was there a reason for changing the sequence? I cannot say what is the best translation, but the fruitful/red example shows what simple translation of a word without context is missing! It may help to give some extended information in tabular form in the description.en.utf8, e.g. if there are different regional traditions?

Reply:
Namaste @gzotti - 

## Regarding translations

The prior translations (as well as wrong spellings) came without any provenance - in some cases reflecting contributor shrI tanmoy's unfamiliarity with the sanskrit language. Sanskrit being highly standardized since ancient, does not recognize big regional varieties. In this changelist, we refer to authoritative dictionaries when needed. For example, in case of phAlgunI, a dictionary derivation (from kalpadruma) is given in the comment. In case of dhaniSThA, the translation "most famous" is likely derived from an alternative name for the same, shraviShThA - so that the english rendering did not match the traditional name. As to why phAlgunI would be "fruitful" or "maghA" bountiful would require much speculation based on vedic rituals and lore - a succinct context is not handily available.

Change in sequence is an accident of moving items to and fro between asterisms and nakshatra-s in the course of this CL. I will restore the original order.

Reply:
(Fixed order.)